test/components/views/rooms/RoomHeader/VideoRoomChatButton-test.tsx

```diff
@@ -80,20 +80,6 @@ describe("<VideoRoomChatButton />", () => {
         jest.restoreAllMocks();
     });
 
-    it("does not render button when room is not a video room", () => {
-        const room = makeRoom(false);
-        getComponent(room);
-
-        expect(screen.queryByLabelText("Chat")).not.toBeInTheDocument();
-    });
-
-    it("renders button when room is a video room", () => {
-        const room = makeRoom();
-        getComponent(room);
-
-        expect(screen.getByLabelText("Chat")).toMatchSnapshot();
-    });
```

Comment:
Initially the button itself was responsible to render itself in case its a video room. The logic now got moved into the RoomHeader itself and the button does no longer feature this behaviour.